src/main/java/com/ita/if103java/ims/controller/InvitationController.java

```diff
@@ -0,0 +1,21 @@
+package com.ita.if103java.ims.controller;
+
+import com.ita.if103java.ims.service.InvitationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/invite")
+public class InvitationController {
+    private InvitationService invitationService;
+
+    @Autowired
+    public InvitationController(InvitationService invitationService) {
+        this.invitationService = invitationService;
+    }
+
+    @PostMapping("/{id}")
+    public void invite(@PathVariable("id") Long accountId, @RequestBody String email) {
+        invitationService.inviteUser(email, accountId);
+    }
```

Comment:
Pleae add handler for this URL
mail.invitationURL=${mail.main.URL}/invite/registration?emailUUID=